packages/amplify-graphql-api-construct/API.md

```diff
@@ -57,6 +57,14 @@ export interface AddFunctionProps {
     readonly runtime?: FunctionRuntime;
 }
 
+// @public
+export interface AmplifyDynamoDBTableStrategy {
+    // (undocumented)
+    readonly dbType: 'DDB';
+    // (undocumented)
+    readonly provisionStrategy: 'AMPLIFY_TABLE';
+}
```

Comment:
How would I configure for `replacement` instead of `iteration`?

Reply:
The param for enabling replacement is not in scope of this PR. Whether to include that in the strategy or not is still TBD. Mark this as TODO.